plugins/modules/vmware_host_capability_facts.py

```diff
@@ -196,10 +195,16 @@ def gather_host_capability_facts(self):
                 supportedVmfsMajorVersion=[version for version in hc.supportedVmfsMajorVersion],
                 vmDirectPathGen2UnsupportedReason=[reason for reason in hc.vmDirectPathGen2UnsupportedReason],
                 ftCompatibilityIssues=[issue for issue in hc.ftCompatibilityIssues],
-                checkpointFtCompatibilityIssues=[issue for issue in hc.checkpointFtCompatibilityIssues],
                 smpFtCompatibilityIssues=[issue for issue in hc.smpFtCompatibilityIssues],
                 replayCompatibilityIssues=[issue for issue in hc.replayCompatibilityIssues],
             )
+
```

Comment:
Could you read `smpFtSupported` and `smpFtCompatibilityIssues` instead, see https://github.com/vmware/pyvmomi/issues/893#issuecomment-614878035